src/app/ReadHandler.h

```diff
@@ -587,20 +582,27 @@ class ReadHandler : public Messaging::ExchangeDelegate
 
     ManagementCallback & mManagementCallback;
 
+    // TODO (#27675): Merge all observers into one and that one will dispatch the callbacks to the right place.
+    Observer * mObserver = nullptr;
+
     uint32_t mLastWrittenEventsBytes = 0;
 
     // The detailed encoding state for a single attribute, used by list chunking feature.
     // The size of AttributeEncoderState is 2 bytes for now.
     AttributeEncodeState mAttributeEncoderState;
 
+    uint16_t mMinIntervalFloorSeconds        = 0;
+    uint16_t mMaxInterval                    = 0;
+    uint16_t mSubscriberRequestedMaxInterval = 0;
+
     // Current Handler state
     HandlerState mState            = HandlerState::Idle;
     PriorityLevel mCurrentPriority = PriorityLevel::Invalid;
     BitFlags<ReadHandlerFlags> mFlags;
     InteractionType mInteractionType = InteractionType::Read;
 
-    // TODO (#27675): Merge all observers into one and that one will dispatch the callbacks to the right place.
-    Observer * mObserver = nullptr;
+    // TODO: We should shutdown the transaction when the session expires.
```

Comment:
```suggestion
```
I don't think that's true.  For a subscription, session-hopping should be absolutely allowed, and a bunch of things actually depend on it happening.  For a read, the exchange will terminate and the right things will happen.

I know, this mis-information comment was pre-existing....

Reply:
Ok. I did not try to fix anything more than mechanically updating the code to the new style. Easy to delete a comment though.